src/Basis/Constructors.jl

```diff
@@ -127,17 +299,18 @@ truepoints = [-1.0, -√(3/7), 0.0, √(3/7), 1.0];
 @assert truepoints ≈ quadraturepoints
 
 # generate Gauss-Legendre-Lobatto points and weights
-quadraturepoints, quadratureweights = LFAToolkit.gausslobattoquadrature(5, true);
+mapping = sausagetransformation(9);
+quadraturepoints, quadratureweights = LFAToolkit.gausslobattoquadrature(5, true, mapping=mapping)
 
 # verify
-trueweights = [1/10, 49/90, 32/45, 49/90, 1/10];
-@assert trueweights ≈ quadratureweights
+@assert quadraturepoints ≈ [-0.9999999999999999, -0.5418159129215785, 0.0, 0.5418159129215785, 0.9999999999999999]
+@assert quadratureweights ≈ [0.18690312494113656, 0.5445666710618016, 0.5400742149974571, 0.5445666710618016, 0.18690312494113656]
```

Comment:
The comment is wrong (these are mapped points), and also, we shouldn't remove the check for unmapped quadrature weights. I would split this into separate tests, first with the standard weights, then with mapping.